src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.service.ProcessorService;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.impl.ProcessorServiceImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import core.basesyntax.strategy.TransactionStrategy;
+import core.basesyntax.strategy.TransactionStrategyImpl;
+import core.basesyntax.strategy.transaction.BalanceTransactionHandler;
+import core.basesyntax.strategy.transaction.PurchaseTransactionHandler;
+import core.basesyntax.strategy.transaction.ReturnTransactionHandler;
+import core.basesyntax.strategy.transaction.SupplyTransactionHandler;
+import core.basesyntax.strategy.transaction.TransactionHandler;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        Map<String, TransactionHandler> transactionHandlersMap = new HashMap<>();
+        transactionHandlersMap.put("b", new BalanceTransactionHandler());
+        transactionHandlersMap.put("s", new SupplyTransactionHandler());
+        transactionHandlersMap.put("p", new PurchaseTransactionHandler());
+        transactionHandlersMap.put("r", new ReturnTransactionHandler());

```

Comment:
actually, it would be better to store these values as an enum, but let it be) I like your style!